product_docs/docs/pem/8/tuning_performance/tuning_wizard.mdx

```diff
@@ -8,84 +8,88 @@ redirects:
 
-The selected actions will apply to all of the changes noted on the Tuning Changes Summary. If you opt to generate a report, PEM will create a report that contains a list of the current values and recommended modifications to the configuration parameters selected on the Tuning Changes Summary dialog. Note that to implement changes, you will need to invoke the Tuning Wizard a second time, specifying the parameters you wish to modify on the `Tuning Changes Summary` dialog.
+PEM makes the recommended changes that you selected in the **Tuning Changes Summary** window. If you choose to generate a report, then PEM creates a report. It contains a list of the current values and recommended modifications to the configuration parameters as selected in the **Tuning Changes Summary** window. To implement changes, you must open the Tuning Wizard a second time, selecting the parameters you want to modify in the **Tuning Changes Summary** window.
 
-Select `Schedule changes` to view and specify your scheduling options.
+Select **Schedule changes** to view and specify your scheduling options.
 
 ![The Schedule or Run? dialog](../images/tuning_wiz_apply_changes.png)
 
-You can:
+The **Configuration now?** slider can be set to:
 
--   Set the `Configuration now?` slider to `Yes` to apply the tuning wizard's recommendations and restart the server now.
--   Set the `Configuration now?` slider to `No` to enable the Time? field and use the calendar selector to specify a time for PEM to apply the tuning wizard's recommendations and restart the server. Note that if you schedule a time for the changes to be applied, you will not be provided with a preview of the change recommendations.
+   - **Yes** &mdash; Apply the Tuning Wizard's recommendations and restart the server.
 
-Select `Generate report` to view your report options.
+   - **No** &mdash; Enables the Time? field where you can specify a date and time with the calendar selector. PEM applies the recommended changes and restarts the server at this time. If you schedule a time for the changes to be applied, then you aren't provided with a preview of the recommended changes.
+
+Select **Generate report** to view your report options.
 
 ![The Schedule or Run? dialog](../images/tuning_wiz_generate_report.png)
 
-You can:
+The **View report now?** slider can be set to:
+
+   - **Yes** &mdash; Display the Tuning Wizard report onscreen.
 
--   Set the `View report now?` slider to `Yes` to display the Tuning Wizard report onscreen.
--   Set the `View report now?` slider to `No` to enable the `Save the report to file` field and use the calendar selector to specify a file name and location to which PEM will write the Tuning Wizard report.
+   - **No** &mdash; Enables the Save the report to file field where you can specify a file name and location. PEM saves the Tuning Wizard report with this name and location.
```

Comment:
Same edit about parallelism and bolding the control. I don't think you need the second sentence.